cmd/api/generic.go

```diff
@@ -148,6 +150,26 @@ func makeTokenRoute(router gin.IRouter, api blockatlas.Platform) {
 	})
 }
 
+func makeCoinRoute(router gin.IRouter) {
+	router.GET("/coins", func(c *gin.Context) {
+		c.JSON(http.StatusOK, coin.Coins)
+	})
+
+	router.GET("/coin/:id", func(c *gin.Context) {
```

Comment:
what is the purpose for `/coin/:id` ? I think `router.GET("/coins"` might be enough

Reply:
let's only add method that we will need

Reply:
I going to remove, I did use in the integrations test, but it is no more useful